docs/domains/index.html

```diff
@@ -1147,18 +1172,23 @@
           "description": "Experimental feature configurations.",
           "$ref": "#/definitions/Experimental"
         },
-        "opssWalletSecret": {
-          "description": "opss key passphrase.",
-          "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.SecretReference"
-        },
         "serverService": {
           "description": "Customization affecting ClusterIP Kubernetes services for WebLogic Server instances.",
           "$ref": "#/definitions/ServerService"
         },
         "domainHome": {
-          "description": "The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeInImage is false. Defaults to /u01/oracle/user_projects/domains/ if domainHomeInImage is true.",
+          "description": "The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeSourceType is not Image. Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType is Image.",
           "type": "string"
         },
+        "domainHomeSourceType": {
+          "description": "Domain home source type: Legal values: Image, PersistentVolume, FromModel. Defaults to Image unless configuration.model is provided and then defaults to FromModel.",
```

Comment:
Same comment as above - maybe just default to Image instead of having a dynamically changing default.